docs/content/contribute/guide.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Contributing Guide
+layout: sources
+type: sources
+aliases:
+- /contributing/
+---
+
+{{% readfile file="/sources/CONTRIBUTING.md" markdown="true" %}}
```

Comment:
A few links in the generated contributing guide don't appear to work for me -- though I can't recall if it's due to an issue of viewing the docs in the PR preview...

For instance, the `Contributors` link of https://deploy-preview-1363--porter.netlify.app/CONTRIBUTORS.md and the `Contribution Ladder` link of https://deploy-preview-1363--porter.netlify.app/CONTRIBUTION_LADDER.md

Otherwise, if they work for you and/or are expected to work after the docs are published, nothing to see here!